manifests/1.3.0/opensearch-dashboards-1.3.0-test.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: OpenSearch Dashboards
+components:
+  - name: functionalTestDashboards
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+schema-version: '1.0'
```

Comment:
I think we like putting the schema-version on line 2.

Reply:
updated